media-sound/amarok/amarok-9999.ebuild

```diff
@@ -15,7 +15,10 @@ LICENSE="GPL-2"
 SLOT="5"
 IUSE="ipod lastfm mariadb mtp ofa podcast wikipedia"
 
-# ipod requires gdk enabled and also gtk compiled in libgpod
+# USE="ipod" requires gdk enabled and also gtk compiled in libgpod.
+# The 'dev-db/mariadb[embedded]' is unavailabe now in portage tree
+# therefore for external MariaDB support the 'dev-db/mariadb-connector-c'
+# is sufficient for DEPEND to build. 'dev-db/mariadb' could be installed later.
```

Comment:
Strictly speaking `dev-db/mariadb[embedded]` was never available in Gentoo. I don't see any information here that isn't conveyed through dependencies and later in `pkg_postinst` anyway, so you can drop that comment.